### PR TITLE
Automation editor: clarify multiple triggers logic

### PR DESCRIPTION
**Pull request by mdimjasevic**
## Proposed change

This PR introduces a small but impactful change to the automation editor within the frontend. It adds a clarification to inform users that the automation will be executed when any of the configured triggers fire.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

There is nothing to configure; this is shown by default when starting out with a new automation.

```yaml

```

## Additional information

I was not aware how multiple triggers are combined in determining whether an automation should fire. Like it currently is, it can be an OR function, but it might have been an AND function or any other logical function that decides from multiple triggers. This note helps to understand that it is sufficient for any trigger to succeed, which alleviates the confusion the user might have in how is the total outcome computed.

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by mdimjasevic** [comment]
When I follow the link for the requested changes, I see nothing but my own proposed changes. Am I missing anything?

**Comment by karwosts** [comment]
> When I follow the link for the requested changes, I see nothing but my own proposed changes. Am I missing anything?

No, you are good now. Nothing left to do but wait for review.

**Review by codyc1515 — COMMENTED** [review]
This whole page is pretty ambiguous for someone who tends to see things in only black and white, with no shades of grey. Nowhere do we clarify when exactly an `and` / `or` / `not` statement is required under the `And if` or `Then do` sections.

To your PR: Personally feel it would be better to say _what_ it is we are talking about first, then when it is used, i.e. a trigger is this, and does this. [The documentation](https://rc.home-assistant.io/docs/automation/trigger/) does a good job of this but it's a bit jargony.

File: src/translations/en.json [hunk 1]
```diff
@@ -2478,7 +2478,7 @@
             "triggers": {
               "name": "Triggers",
               "header": "When",
-              "description": "This list of triggers is what starts your automation. A trigger is a specific event happening in or around your home, for example: ''When the sun sets''.",
+              "description": "This list of triggers is what starts your automation, and any listed trigger can start it. A trigger is a specific event happening in or around your home, for example: ''When the sun sets''.",
```
Comment (jlpouffier) [inline src/translations/en.json:2481]: I checked with our technical writer @c0ffeeca7 
We do not want to make these sentences any longer than what they already are.
We would be ok to change the whole sentence to:

> A trigger is a specific event happening in or around your home, for example: ''When the sun sets''. Any trigger listed here will start your automations

**Review by jlpouffier — CHANGES_REQUESTED** [review]

File: src/translations/en.json (hunk 1 above)
Comment (codyc1515) [reply src/translations/en.json:2481]: That’s what I had in mind.

Comment (mdimjasevic) [reply src/translations/en.json]: Thank you, @jlpouffier ! I've put that sentence by the technical writer in.

Maybe nit-picky, but wouldn't it be better if it was singular "automation" at the end of that sentence given that the user interface provides a way to edit only one automation at a time?

**Review by jlpouffier — APPROVED** [review]

**Review by jlpouffier — APPROVED** [review]

File: src/translations/en.json [hunk 2]
```diff
@@ -2478,7 +2478,7 @@
             "triggers": {
               "name": "Triggers",
               "header": "When",
-              "description": "This list of triggers is what starts your automation. A trigger is a specific event happening in or around your home, for example: ''When the sun sets''.",
+              "description": "A trigger is a specific event happening in or around your home, for example: ''When the sun sets''. Any trigger listed here will start your automations.",
```
Comment (jlpouffier) [inline src/translations/en.json:2481]:
```suggestion
              "description": "A trigger is a specific event happening in or around your home, for example: ''When the sun sets''. Any trigger listed here will start your automation.",
```

@mdimjasevic my bad you're right. "Automations" plural here makes no sense. 
Sorry about that.

**Review by jlpouffier — CHANGES_REQUESTED** [review]

**Review by bramkragten — APPROVED** [review]

**Pull request merged**